appserver/ejb/ejb-container/src/main/java/org/glassfish/ejb/admin/cli/ListTimers.java

```diff
@@ -155,6 +155,11 @@ public void execute(AdminCommandContext context) {
         }
         if (nonNull(ejbTimerService)) {
             result = ejbTimerService.listTimers(serverIds);
+        } else {
+            result = new String[serverIds.length];
+            for (int i = 0; i < result.length; i++) {
+                result[i] = "0";
+            }
```

Comment:
From "Obscure standard library methods" I present you `Arrays.fill(Object[], Object value)`